jb-manager-bot/jb_manager_bot/data_models.py

```diff
@@ -59,4 +62,6 @@ class FSMOutput(BaseModel):
     menu_selector: Optional[str] = None
     menu_title: Optional[str] = None
     form_token: Optional[str] = None
-    plugin_uuid: Optional[str] = None
\ No newline at end of file
+    plugin_uuid: Optional[str] = None
+    retriever_collection_name: str = "KB_Law_Files"
```

Comment:
Can we make it optional and not hard code the values?

Reply:
Cool, but making it optional can give it a None value, which is not allowed as collection name in RAGInput data model. So went with this way. Please let me know what else works.

Reply:
We can't make this optional. Don't supply a default value. That's incorrect. Just keep it:

```python
retriever_collection_name: str
```

Reply:
If the retriever_collection_name is not made optional and does not have a default value, all the FSMOutput object definitions will require this value as it will become a required field.